### PR TITLE
allow to set an icon color different than the text color

### PR DESCRIPTION
**Pull request by e-krebs**
`mmp-icon-color` was set to the same variable (`mini-media-player-base-color`) as `mmp-base-color` and `mmp-text-color` whereas not having the same fallback color (`paper-item-icon-color` vs. `primary-text-color` for both other).
this PR create a `mini-media-player-icon-color` that allows to consistently set an icon color different from those of base-color and text-color.

### CODE REVIEW

File: src/style.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const style = css`
     --mmp-media-cover-info-color: var(--mini-media-player-media-cover-info-color, --mmp-text-color);
     --mmp-text-color-inverted: var(--disabled-text-color);
     --mmp-active-color: var(--mmp-accent-color);
-    --mmp-icon-color: var(--mini-media-player-base-color, var(--paper-item-icon-color, #44739e));
+    --mmp-icon-color: var(--mini-media-player-icon-color, var(--paper-item-icon-color, #44739e));
```
Comment (kalkih) [inline src/style.js]: I would be ok with this change if you make the initial fallback `mini-media-player-base-color` and then  `paper-item-icon-color` etc...

**Comment by kalkih** [comment]
Great, we also need to update the readme with the new theme variable.

**Comment by e-krebs** [comment]
I did both css fallback and updating the README.
This should be ready for review again :)

File: src/style.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ const style = css`
     --mmp-media-cover-info-color: var(--mini-media-player-media-cover-info-color, --mmp-text-color);
     --mmp-text-color-inverted: var(--disabled-text-color);
     --mmp-active-color: var(--mmp-accent-color);
-    --mmp-icon-color: var(--mini-media-player-icon-color, var(--paper-item-icon-color, #44739e));
+    --mmp-icon-color: var(--mini-media-player-icon-color, --mini-media-player-base-color, var(--paper-item-icon-color, #44739e));
```
Comment (kalkih) [inline src/style.js]: I think we have to modify this line a bit for it to work properly, we should probably nest the fallbacks (if more than one), see [this](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#Custom_property_fallback_values).

```js
--mmp-icon-color: var(--mini-media-player-icon-color, var(--mini-media-player-base-color, var(--paper-item-icon-color, #44739e)));
```

**Comment by kalkih** [comment]
Rest looks good, thanks!

File: src/style.js (hunk 2 above)
Comment (e-krebs) [reply src/style.js]: my bad, sorry!
fixed in a new commit

Comment (kalkih) [reply src/style.js]: The nested syntax is pretty ugly and messy imo, the way you wrote it first does actually make more sense imo 🙂

**Comment by kalkih** [comment]
Thanks for the PR, now merged!

**Pull request merged**